Library/Formula/augeas.rb

```diff
@@ -12,7 +12,6 @@ class Augeas < Formula
 
   head do
     url "https://github.com/hercules-team/augeas.git"
-
```

Comment:
I'm the most anal person in the world about this, but not having an empty line between the `url` and the first `depends_on` drives me up the wall :smile_cat:. Feel free to ignore me though.